ui/src/components/timetables/vehicle-schedule-details/VehicleRouteTimetables.tsx

```diff
@@ -1,17 +1,16 @@
-import { RouteWithJourneyPatternStopsFragment } from '../../../generated/graphql';
 import { VehicleRouteTimetableSection } from './VehicleRouteTimetableSection';
 
 interface Props {
-  routes: RouteWithJourneyPatternStopsFragment[];
+  routeIds: UUID[];
 }
 
-export const VehicleRouteTimetables = ({ routes }: Props): JSX.Element => {
+export const VehicleRouteTimetables = ({ routeIds }: Props): JSX.Element => {
```

Comment:
it's not necessarily in the scope of this PR, but this is a pretty bad name for these components. Would these be better?
VehicleRouteTimetables -> LineTimetables
VehicleRouteTimetable -> RouteTimetables